.github/workflows/dockerhub-released-image.yml

```diff
@@ -32,6 +34,8 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: 1.19.5
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.0.3
```

Comment:
sorry, I found the newest cosign is [2.0.2](https://github.com/sigstore/cosign/tree/v2.0.2). Can you give me the url?

Reply:
This is cosign-installer version and not cosign version. github.com/sigstore/cosign-installer

Reply:
Thanks, It goo.
And I think this feature Timely make up for karmada shortcomings in image verification.

Can you explain how to use in customer env in doc？

Reply:
> Can you explain how to use in customer env in doc？

Check links from issue https://github.com/karmada-io/karmada/issues/3435 .

- https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-artifacts/

Reply:
@zishen  I'm change cosign version to v1.13.1 for Consistent behavior with https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-artifacts/ . Thanks for your check.